terraform.tfvars.example

```diff
@@ -291,6 +291,40 @@ freeform_tags = {
       environment = "dev"
       role        = "node-pool"
     }
+    node = {
+      environment = "dev"
+      role        = "instance"
+    }
+  }
+}
+
+# defined_tags
+defined_tags = {
+  # vcn, bastion and operator freeform_tags are required
+  # add more freeform_tags in each as desired
+  vcn = {
+    "cn.environment" = "dev"
+  }
+  oke = {
+    cluster = {
+      "cn.environment" = "dev"
+      "cn.role"        = "cluster"
+    }
+    persistent_volume = {
+      "cn.environment" = "dev"
+    }    
+    service_lb = {
+      "cn.environment" = "dev"
+      "cn.role"        = "load balancer"
+    }
+    node_pool = {
+      "cn.environment" = "dev"
+      "cn.role"        = "node-pool"
+    }
+    node = {
```

Comment:
The defined tags specified here are coming through but the ones specified inside the node pool definition are not coming through